ansible/roles/kong-api/defaults/main.yml

```diff
@@ -5127,3 +5127,20 @@ kong_apis:
       config.limit_by: credential
     - name: request-size-limiting
       config.allowed_payload_size: "{{ small_request_size_limit }}"
+
+  - name: createUserVersion3
+    request_path: "{{ user_service_prefix }}/v3/create"
+    upstream_url: "{{ learning_service_url }}/v3/user/create"
+    strip_request_path: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist: "{{ createUserVersion3_ACL | default(['userUpdate']) }}"
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
```

Comment:
do we require medium rate limiting for this API?

Reply:
@SMY-ALTAMASH API does work same as user/v2/create.